codacy/requirements.yaml

```diff
@@ -22,19 +22,20 @@ dependencies:
   repository: https://charts.codacy.com/external
```

Comment:
can you also update the requirements.lock to match these new versions?

Reply:
done